packages/core/types/config.d.ts

```diff
@@ -115,6 +115,7 @@ export interface DefaultThemeMap {
 	background: 'colors'
 	backgroundColor: 'colors'
 	backgroundImage: 'colors'
+	boderImage: 'colors'
```

Comment:
```suggestion
	borderImage: 'colors'
```

Reply:
Thank you for code reviewing. I applied the suggestion changed, feel free to let me know if anything needs to fix.